app/validators/expiry_date_validator.rb

```diff
@@ -0,0 +1,10 @@
+class ExpiryDateValidator < ActiveModel::Validator
+  def validate(record)
+    if record.expires_at && record.expires_at <= DateTime.now
```

Comment:
Dont use Time.now. always use Time.current

You should look up the differrence